webrtc.html

```diff
@@ -1907,7 +1901,9 @@ <h4>Set the configuration</h4>
             <li>If the value of <code><var>configuration</var>.<a data-link-for=
             "RTCConfiguration">rtcpMuxPolicy</a></code> differs from the
```

Comment:
drive-by nit: shouldn't this be "if the value of ... *is set* and differs" here and elsewhere in this section?

Reply:
done as separate commit!